### ISSUE TITLE
 AttributeError: 'DecimalValidator' object has no attribute 'code'

### ISSUE THREAD

**Issue by backsoul** [ForestAdmin/django-forestadmin#73]
## Expected behavior

TODO:  display decimal field without problem.

## Actual behavior

TODO: AttributeError: 'DecimalValidator' object has no attribute 'code'

## Failure Logs

TODO: Please include any relevant log snippets, if necessary.
![Captura de Pantalla 2021-11-19 a la(s) 10 04 27 p  m](https://user-images.githubusercontent.com/65988584/142712426-adefef84-01d4-4b06-b7c3-cf7a47448402.png)

## Context

TODO: My file requirements.txt

aniso8601==7.0.0
asgiref==3.4.1
Beaker==1.11.0
certifi==2021.10.8
cffi==1.15.0
chardet==4.0.0
cryptography==35.0.0
defusedxml==0.7.1
Django==3.2.9
django-cors-headers==3.7.0
django-forestadmin==1.0.5
django-graphql-jwt==0.3.4
django-ipware==3.0.7
ecdsa==0.17.0
future==0.18.2
graphene==2.1.9
graphene-django==2.15.0
graphql-core==2.3.2
graphql-relay==2.0.1
idna==2.10
importlib-metadata==4.6.3
Mako==1.1.6
MarkupSafe==2.0.1
marshmallow==3.14.1
marshmallow-jsonapi==0.24.0
oic==1.2.1
promise==2.3
pyasn1==0.4.8
pycparser==2.21
pycryptodomex==3.11.0
pyjwkest==1.4.2
PyJWT==2.3.0
python-dateutil==2.8.1
python-jose==3.3.0
pytz==2021.3
requests==2.25.1
rsa==4.7.2
Rx==1.6.1
singledispatch==3.7.0
six==1.16.0
sqlparse==0.4.2
text-unidecode==1.3
typing_extensions==4.0.0
urllib3==1.26.7
zipp==3.6.0

* Package Version: Django==3.2.9
* Django Version: django-forestadmin==1.0.5

**Comment by pierulohacker** [comment]
Same problem here!
This is my log

```
Exception in thread django-main-thread:

Traceback (most recent call last):

File "/usr/local/lib/python3.9/threading.py", line 973, in _bootstrap_inner

self.run()

File "/usr/local/lib/python3.9/threading.py", line 910, in run

self._target(*self._args, **self._kwargs)

File "/usr/local/lib/python3.9/site-packages/django/utils/autoreload.py", line 64, in wrapper

fn(*args, **kwargs)

File "/usr/local/lib/python3.9/site-packages/django_forest/management/commands/runserver.py", line 15, in inner_run

Schema.build_schema()

File "/usr/local/lib/python3.9/site-packages/django_forest/utils/schema/__init__.py", line 119, in build_schema

cls.add_fields(model, collection)

File "/usr/local/lib/python3.9/site-packages/django_forest/utils/schema/__init__.py", line 107, in add_fields

f = handle_validations(field, f)

File "/usr/local/lib/python3.9/site-packages/django_forest/utils/schema/validations.py", line 67, in handle_validations

f = handle_validators(field.validators, f)

File "/usr/local/lib/python3.9/site-packages/django_forest/utils/schema/validations.py", line 53, in handle_validators

f = handle_validator(validator, f)

File "/usr/local/lib/python3.9/site-packages/django_forest/utils/schema/validations.py", line 38, in handle_validator

if validator.code in VALIDATORS.keys():

AttributeError: 'DecimalValidator' object has no attribute 'code'
```

**Comment by vamonte** [comment]
@pierulohacker @backsoul  
Fixed in #89 and deployed in the 1.0.19 version.
I close the issue.
